src/connection-monitor-preview/azext_connection_monitor_preview/tests/latest/test_connection_monitor_preview.py

```diff
@@ -184,7 +175,7 @@ def test_nw_connection_monitor_v2_test_configuration(self, resource_group, resou
                  '--protocol Http '
                  '--frequency 90 '
                  '--http-method Get '
-                 '--http-path "/" '
```

Comment:
Why these arguments removed?

Reply:
It causes a problem, so I deleted it.
```
azure.core.exceptions.HttpResponseError: (PathNotSupportedInBothHTTPConfigurationAndAddress) Path should be specified either in the URI in the endpoint address or in the 'Path' property of HTTPConfiguration. It can't be specified in both places.
```

Reply:
@00Kai0 please run live tests for the updated tests.